build-carthage.sh

```diff
@@ -1,76 +1,22 @@
 #!/bin/bash
 
+# TODO: Remove the script in Certificate repo
```

Comment:
can this be removed now?

Reply:
This is a reminder to remove the scripts we have in Certificate repo for extracting the URL from AudioEngine.json. Once we confirm the AudioEngine 6.5.2 works, I will remove this and the scripts mentioned.